.eslintrc

```diff
@@ -54,17 +54,19 @@
     "no-restricted-imports": 0,
     "class-methods-use-this": 0,
     "no-use-before-define": 0,
+    "object-curly-newline": 0,
     "react/destructuring-assignment": 0,
     "react/no-access-state-in-setstate": "error",
     "react/jsx-filename-extension": 0,
     "react/jsx-indent": 0,
     "react/jsx-indent-props": 0,
+    "react/forbid-prop-types": 0,
```

Comment:
Please create `PropTypes.shape` instead of using `PropTypes.object`

Reply:
https://github.com/Wikia/react-common/pull/22/commits/6327fae41b68b043ce0805639660b3cc3620ebd9